### ISSUE TITLE
[BUG]: MSVC always renders as if it printed a warning

### ISSUE THREAD

**Issue by Alcaro** [compiler-explorer/compiler-explorer#4809]
### Describe the bug

Every time you compile anything with MSVC, the checkmark turns orange, as if there's a warning

### Steps to reproduce

Simply switch to any MSVC. Wine or not doesn't matter, they do the same.

### Expected behavior

Special case MSVC, and/or its output format, so the checkmark remains green

There's an attempt to do that at https://github.com/compiler-explorer/compiler-explorer/blob/main/static/compiler-service.ts#L357, but it's clearly not working.

Originally found at https://discord.com/channels/763778740345765900/764131312117743656/1080832801538060329

### Reproduction link

https://godbolt.org/z/3qYr81vdj

### Screenshots

Not applicable

**Comment by jeremy-rifkin** [comment]
Non-wine msvc seems ok, not sure if / when this behavior changed

**Comment by RubenRBS** [comment]
There's definetely some special-case for the checkmark for msvc compilers yes :)
